index.html

```diff
@@ -130,33 +130,30 @@ <h4><a id="new-project-with-dub" class="anchor" href="#new-project-with-dub" ari
             <li>Once a dub project is loaded there is right click menu option to run with dub, or you can use the run config - run with Dub</li>
-            <li>Before running go and configure the DTools in Preferences -&gt; Other Settings -&gt; D Tools</li>
+            <li>Before running go and configure the DTools in Settings -&gt; Languages & Frameworks -&gt; D Tools</li>
           </ul>
 
           <h4><a id="configure-dtools" class="anchor" href="#configure-dtools" aria-hidden="true"><span class="octicon octicon-link"></span></a>Configure DTools</h4>
 
-          <p>The best option is to do and get the following tools and build them according to their github page instructions:</p>
+          <p>The best option is to do and get the following tools:</p>
 
           <ul>
-            <li><a href="http://code.dlang.org/download">dub</a></li>
-            <li><a href="https://github.com/Hackerpilot/DCD">DCD</a></li>
-            <li><a href="https://github.com/Hackerpilot/Dscanner">DScanner</a></li>
-            <li><a href="https://github.com/Hackerpilot/dfmt">Dfmt</a></li>
+            <li><a href="https://github.com/dlang-community/DCD/releases">DCD</a></li>
+            <li><a href="https://github.com/dlang-community/D-Scanner/releases">DScanner</a></li>
+            <li><a href="https://github.com/dlang-community/dfmt/releases">Dfmt</a></li>
           </ul>
 
-          <p>If you put these tools on your path you can go to Preferences -&gt; Other Settings -&gt; D Tools and just click autofind on each of the tools and it will find them. Otherwise you will need to select the correct path to the tool for each one.</p>
-
-          <p>There is a nice blog post about setting up the tools at: <a href="http://www.samael.me.uk/2015/12/d-plugin-for-intellij-idea.html">www.samael.me.uk</a></p>
+          <p>If you put these tools on your path you can go to Settings -&gt; Languages & Frameworks -&gt; D Tools and just click "Auto Find" on each of the tools and it will find them. Otherwise you will need to select the correct path to the tool for each one.</p>
 
           <h4><a id="configuring-dcd" class="anchor" href="#configuring-dcd" aria-hidden="true"><span class="octicon octicon-link"></span></a>Configuring DCD</h4>
 
-          <p>To configure DCD in the dcd-server add a comma separated list of paths that point to your libraries that you want to include in autocompletion.</p>
+          <p>On MacOS and Windows the path to Phobos and DRuntime is inferred by the D SDK path. Dub dependencies can be passed to DCD by pressing the menu item "Tools -> Process D Libraries". Additional libraries can b passed to DCD by providing a comma separated list of paths in the dcd-server flags field.</p>
 
           <p>For example:</p>
 
-          <pre><code>/Library/D/dmd/src/phobos,/Library/D/dmd/src/druntime/import,/Users/hendriki/.dub/packages/rainbow-master/src</code></pre>
+          <pre><code>/Users/hendriki/projects/rainbow-master/src</code></pre>
```

Comment:
I think the point of the code block was to give an example of the comma separated paths to various D sources.  Now that the preceding text has changed it may be best to show a screenshot of how the _process D libs_ has changed the project structure

Reply:
As a user I am quite not happy about the special behavior of the DCD flags field. I tried several days to set the DCD arguments --port and -I within this field and always wondered why these arguments doesn't work.

I already thought about creating an issue for this. The flags field should work like all other flag fields. Maybe we can add an additional UI field which covers the actual functionality (splitting  the paths at comma character and add as source directories with -I).

Reply:
Unfortunately I do not know what process d Libs exactly does, only that I need to call it from time to time. I changed the line back and describe here the linux case where you have to add Phobos and DRuntime path.